### PR TITLE
Travel pay content updates.

### PR DESCRIPTION
**Pull request by brianseek**
## Description
Content updates for Travel Pay screens

## Original issue(s)
department-of-veterans-affairs/va.gov-team#47705

## Testing done
Added some cypress screenshots for confirmation pages. Updated existing cypress tests.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/check-in/tests/e2e/pages/TravelPages.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class TravelPages {
   validateContent = page => {
     let body = true;
     const helpText = true;
-    if (page === 'vehicle') {
+    if (page === 'vehicle' || 'mileage') {
```
Comment (wullaski) [inline src/applications/check-in/tests/e2e/pages/TravelPages.js:33]:
```suggestion
    if (page === 'vehicle' || page === 'mileage') {
```

File: src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx [hunk 2]
```diff
@@ -91,12 +92,17 @@ const CheckInConfirmation = props => {
               data-testid="travel-pay-message"
             >
               <div>
-                <strong>{t('reimbursement-claim-number')}</strong>
-                <br />
-                {travelPayClaimData.claimId}
+                {t('check-travel-claim-status')}
                 <br />
```
Comment (wullaski) [inline src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx:96]: Could we wrap things in paragraph tags instead?

**Review by wullaski — CHANGES_REQUESTED** [review]

File: src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx (hunk 2 above)
Comment (brianseek) [reply src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx:96]: yes that is a better idea! I got caught up in using what was already there. Great catch

**Review by wullaski — APPROVED** [review]
LGTM!

**Review by acrollet — APPROVED** [review]
Running well for me locally, nice work!

**Pull request merged**